README.md

```diff
@@ -497,9 +497,11 @@ provide a kaniko cache warming image at `gcr.io/kaniko-project/warmer`:
 
 ```shell
 docker run -v $(pwd):/workspace gcr.io/kaniko-project/warmer:latest --cache-dir=/workspace/cache --image=<image to cache> --image=<another image to cache>
+docker run -v $(pwd):/workspace gcr.io/kaniko-project/warmer:latest --cache-dir=/workspace/cache --dockerfile=<path to dockerfile>
 ```
 
-`--image` can be specified for any number of desired images. This command will
+`--image` can be specified for any number of desired images. `--dockerfile` can 
+be specified for the path of dockerfile for cache.These command will combined to 
```

Comment:
nit: can you also add that an optional `--build-arg` flag is available as well w/ this change